recipe/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "streamlit-navigation-bar" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit-navigation-bar-{{ version }}.tar.gz
+  sha256: f458c3f9bbb72a3fa07a28c1db98eceabdd49b19f71fafb44dc36042e0d4b4c4
+
+build:
+  entry_points:
+    - streamlit-navigation-bar = streamlit_navigation_bar:print_version
+  skip: true  # [s390x or py<=38]
```

`setup.py` says `python_requires=">=3.8"`:

```suggestion
  skip: true  # [s390x or py<38]
```

---

It seems like removing the `=` breaks the build.